docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_shell.rst

```diff
@@ -113,7 +113,7 @@ k-points
 ~~~~~~~~
 
 A set of k-points in the Brillouin zone is represented by an instance of the
-``KpointsData`` class. Choose one from the graph of Section :ref:`aiidagraph`,
+``KpointsData`` class. Choose one from the graph of :numref:`aiidagraph`,
```

Comment:
The link `aiidagraph` is actually a section link, not the figure. If you want to reference the figure use `fig_graph`